src/mudlet.cpp

```diff
@@ -3862,15 +3888,18 @@ void mudlet::setShowIconsOnMenu(const Qt::CheckState state)
     }
 }
 
-// during Mudlet load, darktheme will already be set to 'true' -
-// so pass a separate flag to override the state check
-void mudlet::setDarkTheme(const bool& state, const bool& loading)
+void mudlet::setAppearance(const Appearance state, const bool& loading)
 {
-    if (mDarkTheme == state && !loading) {
+    if (state == mAppearance && !loading) {
```

Comment:
Possible logic error here?

|Wanted and Current State are the same|Loading|Run the rest of this method|
|-|-|-|
|No|No|Yes|
|No|Yes|Yes|
|Yes|No|No|
|Yes|Yes|Yes|

Ah, no, that is the right logic...

😌